Kickstarter-iOS/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSUserTrackingUsageDescription</key>
+	<string>We use personal data to provide a good experience on Kickstarter, and to help connect you with projects you&apos;ll love.</string>
```

Comment:
I'm not sure if descriptions in the plist are translated for us. Do you know?

Reply:
good question! Seems like the system itself would handle that, can you check? I know we have some other popups on first launch too..wonder if they have translations.